production/helm/templates/promtail/configmap.yaml

```diff
@@ -58,7 +58,7 @@ data:
         - action: drop
           regex: ^$
           source_labels:
-          - __meta_kubernetes_pod_label_app
+          - __meta_kubernetes_pod_label_name
```

Comment:
If I read this properly, this will `drop` anything with an empty name label.  Do you mean `keep`?